### ISSUE TITLE
Quote from another page..... with a refresh.

### ISSUE THREAD

**Issue by jasonjac2** [stevencohn/OneMore#1192]
# Problem to Solve
I hate duplicating information in multiple places.  I have to put a a note and a link saying "this information is duplicated <link> keep it in sync".

# New Feature/Solution

It would be good to have a plugin where you link to a para and include the "end token" which is basically any text an a refresh link. 

When you refresh it, it grabs everything from the link end point to the text and inserts that immediately below the "quote command".  You can refresh this at any point by pressing the refresh button.

If the text is identical, nothing is changed.

## Alternative Solutions

As I said in the 1st line, I have a manual way of doing this.

# Additional Context

Provide any other context or screenshots about the feature request here.

THE FINE PRINT This is a hobby project that I started for myself. I'm happy to take requests and will promise to
consider each one carefully but offer no guarantees that I will ultimately agree to their
usefulness or have time to implement any of them. If OneNote offers a "close enough" work-around
then I will probably reject the request - you've been warned!

**Comment by stevenmcohn** [comment]
So unlike the Embed Page command, you want to target just a specific paragraph?

A little background, the reason I did not do this initially and instead expanded it to the contents of an entire page is that a paragraph is somewhat vague. It's a fragile entity, easily broken up by carriage returns or even cut/paste. It could perhaps target a table, even a single-cell table like the OneMore Text Box. But again, I'm worried that it's too easily corrupted. Thoughts?

**Comment by jasonjac2** [comment]
yep, fuzziness is the hazard.  So either go for something strict, like </Quote>, but I think "until <whateverTheyPutInHere>.  It is fragile and I haven't used it in practice.
Quote:<link> until "." refresh
Quote:<link> until "newLine" refresh
Quote:<link> until "some Text" refresh

So if it hits the end of the container without hitting the end terminator you get an error.
Maybe I haven't thought it through enough, maybe it should be more closely linked to the outline structure.

**Comment by stevencohn** [comment]
Just thinking.

```
#start-snippet
<my content here...>
#end-snippet
```

**Comment by stevencohn** [comment]
Closing this... to be covered by #2198
